server/src/main/java/keywhiz/service/resources/automation/v2/SecretResource.java

```diff
@@ -482,6 +486,47 @@ public SecretDetailResponseV2 secretInfo(@Auth AutomationClient automationClient
   }
 
   /**
+   * Retrieve contents for a set of secret series
+   *
+   * @excludeParams automationClient
+   *
+   * @responseMessage 200 Secret series information retrieved
+   * @responseMessage 404 Secret series not found
+   */
+  @Timed @ExceptionMetered
+  @POST
```

Comment:
I don't think we need to get fancy with http verbs.  Just make everything a POST.